### ISSUE TITLE
Cannot add disease for newer OMIM id's

### ISSUE THREAD

**Issue by pnrobinson** [monarch-initiative/HpoCaseAnnotator#115]
I would like to add 620155, Rabin-Pappas syndrome
but it is not recognized by HCA and it does not let me add the disease.
Either we need to pull the ids from Monarch API or we need to allow "unknown" ids.

**Comment by ielis** [comment]
Yeah, we're using the OMIM file in the backend that is kind of arcane. 

For the short-term, I recommend to hand edit the resulting JSON file. It works with the rest of the app.

In the long-term, we have to find a way how to address this in future, either Monarch API, or HPOA, or allow free text.
